.gitignore

```diff
@@ -30,3 +30,8 @@ testdir/
 hostkey.ser
 *.h2.db
 *.factorypath
+
+
+automation-credentials.properties
+know_hosts.properties
+know_hosts_error.properties
```

Comment:
I think the right names will be known_hosts and known_hosts_error because they are not property files.